.github/workflows/codespell.yml

```
@@ -0,0 +1,18 @@
+name: CodeSpell
+on:
+  - pull_request
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+jobs:
+  codespell:
+    name: CodeSpell
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: CodeSpell
+        uses: codespell-project/actions-codespell@master
```

Since the spelling does not change over time, I prefer specifying `v2` as the usage.

https://github.com/codespell-project/actions-codespell?tab=readme-ov-file#usage

```suggestion
        uses: codespell-project/actions-codespell@v2
```